src/main/scripts/src/app/shared/perspectives/documents/documents-perspective-presenter.ts

```diff
@@ -31,7 +31,7 @@ export abstract class DocumentsPerspectivePresenter extends PerspectivePresenter
     super(activatedRoute, componentFactoryResolver);
   }
 
-  public ngAfterViewInit() {
+  public ngOnInit() {
     this.loadPerspectiveComponent(DocumentsPerspective.getPerspective(this.perspective).component);
```

Comment:
This is not a good idea since this overrides parent `ngOnInit()` method. We should rather specify abstract method `loadPerspective()` in `PerspectivePresenter` and implement it here instead of this `ngOnInit()`. This method should also be called at the end of `ngOnInit()` method in `PerspectivePresenter`.